### PR TITLE
fix : changed hover color

### LINKED ISSUE #568: [BUG] focus indicator not visible in dark mode for grey nav links
### Description

Set the site to dark mode. Tab through the site on desktop to the navigation on the left-hand side. Press enter on Frontend to show the categories within then tab through Animations, colors, Design Insp... etc. The focus indicator cannot be seen because it is black on a dark purple background. 

### Screenshots

_No response_

### Any additional information?

This is probably a good first issue so if someone else wants to have a go then feel free. Otherwise, I can fix it.

### What browser are you seeing the problem on?

Chrome

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)

### PR DESCRIPTION
**Pull request by KaKashi1210**
Example: Closes #568

### CODE REVIEW

File: components/SideNavbar/SideNavbarElement.tsx [hunk 1]
```diff
@@ -14,12 +14,13 @@ export const SideNavbarElement = ({ name, url }: SubCategories) => {
...
-          ? "border-l-4 border-violet-500 text-violet-700 dark:text-violet-400"
+          ? "border-l-4 border-violet-500 text-violet-700 dark:text-violet-400 "
           : "text-slate-600 dark:text-slate-400"
-      } collapse w-full text-start pl-3 hover:text-violet-500 dark:hover:text-violet-300 `}
+      } collapse w-full text-start pl-3 hover:text-violet-500 dark:hover:text-violet-300 
+      focus-visible:outline-none focus-visible:ring focus-visible:ring-violet-400 focus-visible:rounded-lg`}
     >
-      <div className="ml-2 text-lg py-2 capitalize transition-all duration-300 hover:pl-2 dark:border-gray-600">
-        {name}
+      <div className="ml-2 text-lg py-2 capitalize transition-all duration-300 hover:pl-2 dark:border-purple-600">
```
Comment (rupali-codes) [inline components/SideNavbar/SideNavbarElement.tsx:22]:
```suggestion
      <div className="ml-2 text-lg py-2 capitalize transition-all duration-300 hover:pl-2 dark:border-violet-500">
```

**Comment by KaKashi1210** [comment]
Done

**Review by CBID2 — APPROVED** [review]
Let's get to merging! :)

**Review by rupali-codes — APPROVED** [review]
great thanks

**Pull request merged**